src/auth/googleauth.ts

```diff
@@ -616,4 +657,63 @@ export class GoogleAuth {
 
     return {client_email: data.default.email};
   }
+
+  /**
+   * Automatically obtain a client based on the provided configuration.  If no
+   * options were passed, use Application Default Credentials.
+   */
+  async getClient() {
+    if (!this.cachedCredential) {
+      if (this.keyFilename) {
+        const filePath = path.resolve(process.cwd(), this.keyFilename);
```

Comment:
I guess it's the same as just `path.resolve(this.keyFilename)`, `process.cwd()` is appended by default

Reply:
I believe you're right :)